### DISCUSSION TITLE
table_master 模型八位格式坐标decode问题

### DISCUSSION THREAD

**Discussion by plotnine1219** [PaddlePaddle/PaddleOCR#12303] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

ppocr/postprocess/table_postprocess.py
这个套件的 185行，x, y, w, h = bbox，没有考虑到八位坐标格式，导致报错

**Comment by GreatV** [comment]
@plotnine1219 是否愿意提一个PR修复来这个问题

**Comment by plotnine1219** [comment]
> @plotnine1219 是否愿意提一个PR修复来这个问题

能力有限解决不了……

**Comment by crackso** [comment]
这个不是TableMasterLabelDecode的嘛，用的是4位坐标

**Comment by plotnine1219** [comment]
> 这个不是TableMasterLabelDecode的嘛，用的是4位坐标

对啊，但是配置文件有8位的选项，到这里就报错了

**Comment by UserWangZz** [comment]
可以简单写一个转换函数嘞～

**Comment by crackso** [comment]
> 可以简单写一个转换函数嘞～

其实也是的，8位转4位是好转的，变成矩形嘛，但8位其实是可以表述变形的。

**Comment by crackso** [comment]
> > 这个不是TableMasterLabelDecode的嘛，用的是4位坐标
> 
> 对啊，但是配置文件有8位的选项，到这里就报错了

训练说明那边是有提示的。配置文件嘛，应该没有具体备注。2者其实是不通的。
